__tests__/app.test.js

```diff
@@ -69,4 +69,25 @@ describe('app', () => {
             })
         });
     });
-});
\ No newline at end of file
+    describe('GET /api/reviews/:review_id', () => {
+        test('200: GET responds status 200 and a review object', () => {
+            return request(app)
+            .get('/api/reviews/1')
+            .expect(200)
+            .then(({body}) => {
+                const review = body
+                expect(review.review_id).toBe(1)
```

Comment:
You can have this expect within the toMatchObject, just put 1 in the place of an expect.any